src/core/token.js

```diff
@@ -1 +1,21 @@
-export const token = "JSESSIONID=521ee879-3b83-4cf9-b6ea-920653197cad";
+/**
+ * This function extract token from COOKIE and returns it
+ *
+ * @author Bogdan Shcherban <bsc@piogroup.net>
+ * @return {string}
+ */
+function getTokenFromCookie() {
+    let result = null;
+    const decodedCookie = decodeURIComponent(document.cookie).split(';');
+    decodedCookie.forEach(item => {
+        let itemArray = item.split('=');
+        let parameterName = itemArray[0];
+        if (parameterName.trim() === "JSESSIONID") {
+            result = itemArray[1];
+        }
+    });
+    return "JSESSIONID=" + result;
+}
+
+export const token = getTokenFromCookie();
+export const domainName = "http://dev.ripple.foundation";
```

Comment:
this should come from a global config variable

Reply:
@fzaninotto 
Could you please clarify, do you mean React context using or simple globar variables like in JS?

Reply:
```
// index.html

<script>
windows.config = {
    mainColor: "#fff"
}
</script>
<script src="/static/main.js" /> 

// in scr/theme.js
const mainColor = windows && windows.config ? global.config.mainColor : '#efe';
```

see https://immutablewebapps.org/